enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -21,14 +20,15 @@
 
 // clang-format off
 #ifdef OCCLUM
-  #include <sgx_report.h>
+#include <sgx_report.h>
 #elif defined(SGX)
-  #include <sgx_urts.h>
-  #include <sgx_quote.h>
+#include <sgx_urts.h>
+#include <sgx_quote.h>
+#include "sgx_stub_u.h"
```

Comment:
Why changing it?

Reply:
I mean the format tool did these changes?

Reply:
sorry, miss the coment. had changed it a lot. 
currently I had rebased yijuan's ecc implementation. And user callback is ongoing, will complete soon. CICD has been finished(my private repo has some problem about CICD test enviroment)